roles/openstack/nova/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+# General defaults
+
+prefix: "{{ provisioner.prefix | default(random.uuid) }}"
+
+# Instances defaults
+
+auto_ip: no
+config_drive: no
+instance_image: centos-minimal
```

Comment:
I think we can assume more users will have centos-minimal or the ability to upload it.